pkg/files/output_file.go

```diff
@@ -11,10 +11,11 @@ import (
 type OutputFile struct {
 	relativePath string
 	data         []byte
+	MarkedType   Type
```

Comment:
might be worth changing MarkedType to private and adding a readonly accessor similar to RelativePath() and Bytes()

Reply:
This has been made not public and is available via a Type() function for OutputFiles.